jetty-util/src/test/java/org/eclipse/jetty/util/RolloverFileOutputStreamTest.java

```diff
@@ -181,6 +181,47 @@ public void testMidnightRolloverCalcSydneyDSTEnd()
         assertSequence(midnight, expected);
     }
 
+    @Test
```

Comment:
If the directory does exist and no exception is thrown we will never enter the `catch` block and the test will pass.

so maybe something like this:
```java
@Test
public void testMissingDirectory()
{
    String templateString = "missingDir/test-rofos-yyyy_mm_dd.log";
    Throwable error;
    try (RolloverFileOutputStream ignored = new RolloverFileOutputStream(templateString))
    {
        throw new IllegalStateException();
    }
    catch (Throwable t)
    {
        error = t;
    }

    assertNotNull(error);
    assertThat(error, instanceOf(IOException.class));
    assertThat(error.getMessage(), containsString("Log directory does not exist."));
}
```

Reply:
I simplified test method as suggested.